src/Build/BackEnd/Components/SdkResolution/SdkResolverLoader.cs

```diff
@@ -36,15 +37,20 @@ internal class SdkResolverLoader
 
         internal virtual IReadOnlyList<SdkResolver> GetDefaultResolvers()
         {
+
+            MSBuildEventSource.Log.SdkResolverLoadResolversStart();
             var resolvers = !string.Equals(IncludeDefaultResolver, "false", StringComparison.OrdinalIgnoreCase) ?
                 new List<SdkResolver> { new DefaultSdkResolver() }
                 : new List<SdkResolver>();
 
+            MSBuildEventSource.Log.SdkResolverLoadResolversStop(string.Empty, resolvers.Count);
```

Comment:
I don't think this is worth measuring. It's just an instantiation of a couple of objects.

Reply:
It also it reports the number of resolvers loaded and without these events the log would not report the correct number of loaded resolvers. But we are probably not super interested in that, so theoretically we can drop them.

Reply:
Yes, we're not really _loading_ any resolvers here so I think we wouldn't miss anything without this event.

Reply:
Agreed, removed them.